gnd/src/sharedTest/java/com/google/android/gnd/FakeData.java

```diff
@@ -60,7 +60,7 @@ public static Survey.Builder newSurvey() {
         .setId("SURVEY")
         .setTitle("Survey title")
         .setDescription("Test survey description")
-        .setAcl(ImmutableMap.of(FakeData.USER.getEmail(), "contributor"));
+        .setAcl(ImmutableMap.of(FakeData.USER.getEmail(), "data_collector"));
```

Comment:
Perhaps roles in the db it should in slug case (eg `data-collector`, dash instead of underscore) for consistency with examples in Firestore docs? This seems to be more idiomatic for the Firestore's JSON-like data structure.

If this is because you plan to add explicit mappings between enum names and db strings in a followup PR please disregard.

Reply:
I think we need the underscore for now for the existing enum name mapping to work.

Do we need to update the mapping to map from "data-collector" -> Role.DATA_COLLECTOR instead of "data_collector" -> Role.DATA_COLLECTOR?

Reply:
> I think we need the underscore for now for the existing enum name mapping to work.

Ok, SGTM.

> Do we need to update the mapping to map from "data-collector" -> Role.DATA_COLLECTOR instead of "data_collector" -> Role.DATA_COLLECTOR?

Yep, once we have the explicit mapping in place.